src/lib/route-data/route-data-function-types.ts

```diff
@@ -0,0 +1,14 @@
+import { L10nTranslationService } from 'angular-l10n';
+import { Data, Params } from '@angular/router';
+import { RouteDataInterface } from './route-data.interface';
+
+/** Type of method that returns a label based on the activated route's params, data and queryParams.
+ * angular-l10n's L10nTranslationService can be used to provide multi-lingual labels */
+export type LabelFunction = (
+    translation: L10nTranslationService,
+    params: Params,
+    routeData: Data,
+    queryParams: Params
+) => string;
+
+export type RouteData = { [path: string]: RouteDataInterface };
```

Comment:
ich meinte eigentlich dass dieser type mit in die Datei _route-data.interface.ts_ kommt.